Makefile

```diff
@@ -47,6 +47,9 @@
 #
 MAKEFLAGS = --no-print-directory
 
+MODULES_TO_LINT = \
+	$(shell find $(CHPL_MAKE_HOME)/modules/dists -name '*.chpl')
+
```

Comment:
This makes me think that `chplcheck` should just accept a list of files or directories, and just recurse directories for `.chpl`?

Reply:
I think I'd prefer the Makefile variable here. Why? Well, there are several cases to consider:

I might want to gather files from only a single directory, without recursing (this may be true if I went through e.g. one sub-folder of `modules/packages`), but recurse in another (as I'm doing with `dist`). I might also want to list a single file. I could achieve all this by specifying files and folders using `--folder` an `--recursive-folder` to `chplcheck`, but it seems cleaner to just use standard tools like `find` than to have to re-implement glob searching in yet another Python script.